rclpy/test/test_subscription.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+import rclpy
+from rclpy.node import Node
+from test_msgs.msg import Empty
+
+
+class TestSubscriber(Node):
+
+    def __init__(self, topic):
+        super().__init__('minimal_subscriber')
```

Comment:
I recommend adding a few remap rules that affect a couple of the test cases to confirm the topic name returned is actually the one being used, and not the original unremapped topic name.